scenarios.md

```diff
@@ -18,9 +18,31 @@ The [OCI TOB][oci-tob] has adopted [OCI Artifacts][artifacts-repo], generalizing
 
 This document serves as the requirements and constraints of a generalized signing solution. It focuses on the scenarios and needs, and very specifically avoids any reference to other projects or implementations. As our working group forms a consensus on the requirements, the group will then transition to a spec.
 
+The following represent the core goals of Notary v2:
+
+1. Registry Native
+   * Signatures and artifacts co-located for easier and secure management
+1. Secure
+   * Attesting to its authenticity and/or certification
+   * No trust on first use, no implicit permissions on rotated keys, secure private keys and PKI
+1. Multiple & Indepdent Signatures
+   * Enabling originating vendor, aggregator certification, customer validation scenarios
+1. Portable
+   * Artifacts move within and across registries supporting provenance, validation and trust
+1. Multi-tenant
+   * Enable cloud providers and enterprises to easily support managed services at scale
+1. Offline & Air-gapped
+   * Artifacts can be signed offline
+   * Artifacts and signatures can be moved into air-gapped environments
+1. Usable
+   * Simple commands to integrate with toolchains, supporting key hierarchies
+   * Signing an artifact must not change the tag or digest enabling existing dev through production workflows
+   * Key acquisition must support users from hobbyists, open source projects to large software vendors
+
 ## Non-Goals
 
-- Notary v2 does not account for what the content represents or its lineage. Other efforts may attach additional content, and re-sign the super set of content to account for other scenarios. 
+1. Notary v2 does not account for what the content represents or its lineage. Other efforts may attach additional content, and re-sign the super set of content to account for other scenarios.
+2. Backwards compatibiliyt with Notay v1
```

Comment:
```suggestion
2. Backwards compatibility with Notary v1
```

Reply:
Thanks. I found another spell checking error. I would have used the [commit suggestion] button, but I was afraid it wouldn't `-s` the commit and I'd be in DCO hell :(